manifests/02-rbac.yaml

```diff
@@ -34,32 +44,44 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - ""
+  resources:
+  - namespaces
+  verbs:
+  - get
```

Comment:
This was moved from the list below.
I'm not really sure if it is necessary, the operator doesn't seem to get any namespace directly, it only ever refers to namespace when interacting with other resources.

Reply:
Aha I missed that, thanks for pointing it out.